### ISSUE TITLE
Support for Arduino Uno R3?

### ISSUE THREAD

**Issue by jacob-newormics** [NewhavenDisplay/NHD-3.12-25664UC_Example#1]
I recently purchased an NHD-3.12-25664UCW2 and found this repository for controlling it with an Arduino Mega. I'm trying to modify it to work with my Arduino Uno R3 (changing the SCK and MOSI ports appropriately) but haven't had success yet. Do you know if it is possible to control the screen with an Uno? If so, do you have any tips or would you be willing to help me troubleshoot?

**Comment by CJoh-NHD** [comment]
Hello Jacob,

I apologize for the delayed response.

To start, I would recommend not only changing `SCK` and `MOSI`, but the other control signals as well.
To something like this:
``` cpp
#define OLED_DC    3    // Register select signal
#define OLED_RESET 4    // Reset signal
#define OLED_CS    5    // Chip select signal
#define OLED_CLK   6    // Serial clock signal
#define OLED_MOSI  7    // Serial data signal
```

Additionally, make sure you are using 3.3V signals to talk to the display. Arduino Uno boards typically use 5V signals, so you may need to use a voltage shifting circuit or an Arduino board that runs on 3.3V.

**Comment by jacob-newormics** [comment]
It's been a while, but I believe I solved this issue by getting the right logic shifting set up as you eventually suggested. The support team (Zach and Mayra) at NHD helped me solve this issue. Since then, I've moved on to using an Arduino Due instead, which runs at 3.3V. This code was great for setting up the initially IO/testing out the look of the display, but for further programming, I've been using the [u8g2 library](https://github.com/olikraus/u8g2).
